modules/era/sql/era_droplist.sql

```diff
@@ -19106,8 +19079,8 @@ INSERT INTO `mob_droplist` VALUES (2401,0,0,1000,4753,@UNCOMMON); -- Scroll Of F
 
 -- ZoneID: 190 - Thousand Eyes
 INSERT INTO `mob_droplist` VALUES (2402,0,0,1000,939,@VCOMMON);   -- Hecteyes Eye (Very Common, 24%)
-INSERT INTO `mob_droplist` VALUES (2402,0,0,1000,4754,@UNCOMMON); -- Scroll Of Fire Iii (Uncommon, 10%)
-INSERT INTO `mob_droplist` VALUES (2402,0,0,1000,1289,@RARE);     -- Burning Hakutaku Eye (Rare, 5%)
+INSERT INTO `mob_droplist` VALUES (2402,0,0,1000,1289,@UNCOMMON); -- Burning Hakutaku Eye (Uncommon, 10%)
+INSERT INTO `mob_droplist` VALUES (2402,0,0,1000,4754,@VRARE);    -- Scroll Of Fire Iii (Very Rare, 1%)
```

Comment:
Fire III might be 5% instead of 1%

https://ffxiclopedia.fandom.com/wiki/Thousand_Eyes
This shows 800 data with around 9%

http://www.ffxidb.com/zones/190/thousand-eyes
This has a lot less data but seems to be more consistent with 5% than 1%

Reply:
Bumped up to 5%